content/index.md

```diff
@@ -1,14 +1,16 @@
 title: complate: declarative, component-oriented HTML templating
+h1: complate
+subtitle: component based templating
+description: expressive • declarative • composable • independent of any particular technology choice
 
-complate
-========
+Modern JavaScript frameworks come with batteries included and everything
+but the kitchen sink.
 
-rendering HTML with expressive, declarative and composable abstractions (i.e.
-components) -- independent of any particular technology choice
+We do **_a lot less_** and we think that is a good thing.
 
-> say what you mean
-
-— [Web Components: The Long Game](https://infrequently.org/2017/10/web-components-the-long-game/)
+complate provides a lightweight [JSX](https://facebook.github.io/jsx/)
+implementation to allow us to write declarative component based templates
+(macros) without making assumptions about the underlying technology.
```

Comment:
I'm not entirely sure that "underlying technology" is easy to understand. What we mean is "independent of your server-side language or framework", right?

Reply:
Yes, absolutely! Thank you 👌